examples/spark2/src/main/java/org/apache/carbondata/examples/sdk/CarbonReaderExample.java

```diff
@@ -61,7 +61,8 @@ public static void main(String[] args) {
             CarbonWriter writer = CarbonWriter.builder()
                 .outputPath(path)
                 .withLoadOptions(map)
-                .withCsvInput(new Schema(fields)).build();
+                .withCsvInput(new Schema(fields))
+                .writtenBy("CarbonReaderExample").build();
```

Comment:
`.build` should be moved to next line also. please follow coding style

Reply:
done, actually initially it was like that, and im using carbon formatting only, but i do not know why it is formatting like this, need to once check the xml